src/state/slices/portfolioSlice/utils.ts

```diff
@@ -170,9 +171,7 @@ export const accountToPortfolio: AccountToPortfolio = args => {
         }
 
         ethAccount.chainSpecific.tokens?.forEach(token => {
-          if (!args.assetIds.includes(token.assetId)) {
-            return
-          }
+          if (!isNft(token.assetId) && !args.assetIds.includes(token.assetId)) return
```

Comment:
we don't have nft assets in the store at the time of portfolio, just add them knowing the nft assets will be dynamically added from tx history after portfolio is loaded

Reply:
added comment for posterity